app/models/article.rb

```diff
@@ -1,5 +1,7 @@
 class Article < ApplicationRecord
-  has_many :comments
+  include Visible
```

Comment:
What exactly do we include?

Reply:
We include exactly the Visible concern, it is a specific module to check if the status of an article is public.